Add Aave DAO proposal validation strategy

pyk opened the pull request:
Changes proposed in this pull request:
- Add Aave proposal validation strategy

ChaituVR reviewed (changes requested):
Hi @pyk Looks like it is same as Basic validation, can you try to use the same on your space than a new validation?

pyk commented:
Hi @ChaituVR,

Thank you so much for the review!

> Looks like it is same as Basic validation, can you try to use the same on your space than a new validation?

I implemented basic validation and added whitelist on top, based on the spec from the AAVE team. You're right, they're dropping this task and looking for an existing solution. I'll close this pull request for now.
